packages/drivers/odsp-driver/src/odspUrlHelper.ts

```diff
@@ -110,8 +110,10 @@ export async function getOdspUrlParts(url: URL): Promise<IOdspUrlParts | undefin
 			}
 		}
 
-		const driveId = joinSessionMatch[3] || joinSessionMatch[5];
-		const itemId = joinSessionMatch[4];
+		// Why are we non null asserting here
+		const driveId = joinSessionMatch[3] || joinSessionMatch[5]!;
```

Comment:
its failing in CI here I need to change it to use ?? instead; https://dev.azure.com/fluidframework/public/_build/results?buildId=275487&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=bfe27d5f-9219-57c0-0d67-1b2946a32921